kubernetes/host.go

```diff
@@ -207,11 +207,11 @@ func HasMatchingVirtualServices(host Host, virtualServices []networking_v1alpha3
 
 // HasMatchingRegistryService returns true when the FDQN of the host param matches
```

Comment:
I'd update the comment to indicate when the namespace parameter is used in the condition.